coordinator/src/main/java/org/apache/uniffle/coordinator/CoordinatorGrpcService.java

```diff
@@ -228,7 +254,8 @@ public void accessCluster(AccessClusterRequest request, StreamObserver<AccessClu
             new AccessInfo(
                 request.getAccessId(),
                 Sets.newHashSet(request.getTagsList()),
-                request.getExtraPropertiesMap()
+                request.getExtraPropertiesMap(),
+                request.getUser()
```

Comment:
Is this a compatible feature? What will happen if a old client request a new coordinator service?

Reply:
Updated in [https://github.com/apache/incubator-uniffle/pull/311/commits/36fb7202f6695eef2c786c65374f5bc5fd6372ef](https://github.com/apache/incubator-uniffle/pull/311/commits/36fb7202f6695eef2c786c65374f5bc5fd6372ef)